src/test/java/uk/gov/companieshouse/api/testdata/service/impl/CompanyProfileServiceImplTest.java

```diff
@@ -47,56 +50,105 @@ class CompanyProfileServiceImplTest {
     @InjectMocks
     private CompanyProfileServiceImpl companyProfileService;
 
-    @Test
-    void createEnglandWales() {
-        final Address mockServiceAddress = new Address(
-                "","","","","",""
-        );
-        CompanySpec spec = new CompanySpec();
+    private Address mockServiceAddress;
```

Comment:
This seems to be used only in `assertCompanyProfile` and maybe could be made a local variable instead? Also it seems to refer to the registered office address, not the service address